cli/azd/pkg/project/project_config.go

```diff
@@ -170,6 +173,20 @@ func ParseProjectConfig(yamlContent string) (*ProjectConfig, error) {
 		)
 	}
 
+	if projectFile.Metadata != nil && projectFile.Metadata.MinSupportedVersion != "" {
+		minSupportedVersion, err := semver.Parse(projectFile.Metadata.MinSupportedVersion)
+		if err != nil {
+			return nil, fmt.Errorf("invalid version for metadata.minSupportedVersion: %w", err)
+		}
+
+		if curVersion, err := semver.Parse(internal.GetVersionNumber()); err == nil {
+			if !internal.IsDevVersion() && curVersion.LT(minSupportedVersion) {
+				return nil, fmt.Errorf("this project requires at least version: %s but you have: %s",
```

Comment:
We should tell them what version of what and tell them how to get it

Reply:
> We should tell them what version of what and tell them how to get it

I will change this to "this project requires at least version: %s of azd but you have: %s. Visit https://aka.ms/azure-dev/install to install the latest version."